client/src/pages/CalendarPage/CalendarPage.module.css

```diff
@@ -0,0 +1,53 @@
+.bg {
+  background-image: linear-gradient(
+    #032567 30%,
+    rgba(195, 221, 253, 30) 60%,
+    #032a5c
+  );
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.calendarHeader {
+  color: white;
+  font-weight: 600;
+  font-family: "Source Sans Pro", sans-serif;
+  font-size: 1.875rem;
+  margin-left: 25px;
+  text-decoration: underline;
+}
+
+.filterButton {
+  font-family: "Source Sans Pro", sans-serif;
+  font-size: 1.125rem;
+  font-weight: 600;
+  background: transparent;
+  border: none;
+  color: #dadada;
+}
+
+.flexContainer {
+  display: flex;
+  justify-content: flex-end;
+  margin: 10px 20px 15px 0px;
+}
+
+ul {
+  margin: 10px 25px;
+  padding: 0px;
+  overflow: hidden;
+  overflow-y: scroll;
+}
+
+/* works in chrome and safari */
+ul::-webkit-scrollbar {
+  width: 0px !important;
+  display: none;
+}
+
+body,
```

Comment:
For the CSS on `body` and `html` I think we can move that to `index.css` (found on the root of the `src` folder

Reply:
ul can also be moved to the `index.css` although one thing to be careful is, since you're directly targeting ALL `ul` elements, then for any subsequent components that use `ul`, these styles are going to be applied

Reply:
Moved `body` and `html` styles moved to `index.css`. Changed from targetting all `ul` to the single task list portion using `taskList` css class.